bip-0003.md

```diff
@@ -0,0 +1,753 @@
+    The comments feature saw insignificant adoption. Few BIPs received any comments and barely any more than two with
+    only a handful of contributors commenting at all. This led to many situations in which one or two comments ended up
+    dominating the comment summary. While some of those comments may have been representative of broadly held opinions,
+    it also overstated the importance of individual comments directly in the Preamble of BIPs. As collecting feedback in
+    this accessible fashion failed, the new process puts the onus back on the audience to make their own evaluation.
+[^layer]: **Why is the layer header now permitted for other BIP types?**  
+    The layer header had already been used by many Informational BIPs, so the rule that it is only available to
+    Standards Track BIPs is dropped.
+[^OtherImplementations]: **What is the issue with "Other Implementations" sections in BIPs?**  
+    In the past, some BIPs had "Other Implementations" sections that caused frequent change requests to existing BIPs.
+    This put an onus on the BIP authors, and frequently led to lingering pull requests due to the corresponding BIPs’
+    authors no longer participating in the process. Many of these alternative implementations eventually became
+    unmaintained or were low-quality to begin with. Therefore, "Other Implementations" sections are heavily discouraged.
+[^complete]: **Why was the Proposed status renamed to Complete?**  
+    Some reviewers of this BIP raised that in a process which outlines the workflow of Bitcoin Improvement _Proposals_
+    using "Proposed" as a status field value was overloading the term: clearly _proposals_ are proposed at all stages of
+    the process. "Complete" expresses that the authors have concluded planned work on all parts of the proposal and are
+    ready to recommend their BIP for adoption. The term "ready" was also considered, but considered too subjective.
+[^rejection]: **Why can proposals remain in Draft or Complete indefinitely?**  
+    The automatic 3-year timeout of BIPs has led to some disagreement in the past and seems unnecessary in cases where
+    the authors are still active in the community and still consider their idea worth pursuing. On the other hand,
+    Draft proposals that appear stale may be tested and cleared out after only one year which should achieve the main goal of
+    the original rule by limiting the effort and attention spent on proposals that never reach Complete.
+[^closed]: **Why was the Closed Status introduced?**  
+    The Closed Status provides value to the audience by indicating which documents are only of historical significance.
+    Previously, the process had Deferred, Obsolete, Rejected, Replaced, and Withdrawn which all meant some flavor of
+    "work has stopped on this". The many statuses complicated the process, may have contributed to process fatigue, and
```

Comment:
```suggestion
    "work has stopped on this." The many statuses complicated the process, may have contributed to process fatigue, and
```

Reply:
How American English places punctuation around subsentences will forever seem odd to me. ^^